sqlalchemy_redshift/dialect.py

```diff
@@ -254,7 +255,9 @@ class RedshiftDDLCompiler(PGDDLCompiler):
     ...     metadata,
     ...     sa.Column('id', sa.Integer, primary_key=True),
     ...     sa.Column(
-    ...         'name', sa.String, info={'distkey': True, 'sortkey': True}
+    ...         'name', sa.String,
```

Comment:
Can we either split these to separate lines, or roll the last two options onto this same line for better consistency?